en/deploy/README.md

```diff
@@ -4,243 +4,301 @@
-     create mode 100644 manage.py
-     create mode 100644 requirements.txt
-     create mode 100644 runtime.txt
 
-## Pick an application name
+### Collecting static files.
+
+We you wondering what "whitenoise" thing was?  It's a tool for serve so-called "static files".  Static files work differently on servers compared to on our own computer, and we need a tool like "whitenoise" to serve them.
+
+We'll find out a bit more about static files later in the tutorial, when we edit the CSS for our site.
+
+For now we just need to run an extra command called "collectstatic", on the server.  It tells django to gather up all the static files it needs on the server.  Mostly, these are the static files that make the admin site look pretty at the moment.
+
+
+    20:20 ~ $ python manage.py collectstatic
+
+    You have requested to collect static files at the destination
+    location as specified in your settings:
+
+        /home/edith/my-first-blog/static
+
+    This will overwrite existing files!
+    Are you sure you want to do this?
+
+    Type 'yes' to continue, or 'no' to cancel: yes
+
+Type "yes", and away it goes!  Don't you love making computers print out pages and pages of impenetrable text?  I always make little noises to accompany it.  Brp, brp brp...
+
+    Copying '/home/edith/.virtualenvs/mvenv/lib/python3.4/site-packages/django/contrib/admin/static/admin/js/actions.min.js'
+    Copying '/home/edith/.virtualenvs/mvenv/lib/python3.4/site-packages/django/contrib/admin/static/admin/js/inlines.min.js'
+    [...]
+    Copying '/home/edith/.virtualenvs/mvenv/lib/python3.4/site-packages/django/contrib/admin/static/admin/css/changelists.css'
+    Copying '/home/edith/.virtualenvs/mvenv/lib/python3.4/site-packages/django/contrib/admin/static/admin/css/base.css'
+    62 static files copied to '/home/edith/my-first-blog/static'.
+
+
+### Creating the database on PythonAnywhere
+
+Here's another thing that's different between your own computer and the server -- it uses a different database.  So, the user accounts and posts can be different on the server and on your computer.
+
+So we initialise the database on the server just like we did the one on your own computer, with `migrate` and `createsuperuser`:
+
+
+    (mvenv)20:20 ~ $ python manage.py migrate
+    Operations to perform:
+    [...]
+      Applying sessions.0001_initial... OK
+
+
+    (mvenv)20:20 ~ $ python manage.py createsuperuser
+
+
+## Publishing our blog as a web app
+
+Now our code is on PythonAnywhere, our virtualenv is ready, the static files are collected, and the database is initialised, we're ready to publish it as a web app.
+
+Click back to the PythonAnywhere dashboard by clicking on its logo, and go click on the **Web** tab, and hit **Add a new web app**.
+
+In the dialog, after confirming your domain name, choose **manual configuration** (NB *not* the "Django" option). Next, choose **Python 3.4**, and click Next to finish the wizard.  
```

Comment:
AS PART OF THE DJANGOGIRLS TUTORIAL YOU MUST ALSO LEARN BASIC LATIN.  DJANGOGIRLS VENIT, DJANGOGIRLS VIDET, DJANGOGIRLS VICIT

on the plus side, you don't need to translate it for different languages ;)

i'll replace it with sthing else if you really want?